### ISSUE TITLE
Option to specify scalafix version number

### ISSUE THREAD

**Issue by jahan01** [cosmicsilence/gradle-scalafix#30]
Would be nice not to depend upon plugin to set the scalafix dependency version.

It would be nice if a property is exposed for this

**Comment by marcelocenerine** [comment]
@jahan01 I believe you are able to override the version of scalafix by adding the following to your build script:

```groovy
buildscript {
    dependencies {
        classpath 'ch.epfl.scala:scalafix-cli_2.12.12:0.9.23'
    }
}
```

**Comment by jahan01** [comment]
@marcelocenerine Thanks for your response and it does work. The premise for me wanting to update the version is the I see some difference between rules applied when running via `gradle scalafix` vs `sbt scalafix`. So I want to update to same version as `sbt scalafix` (scalafix version: 0.9.23 and semanticDb: 4.3.24).

Issues I found are:

1. Certain rules are not applied by `gradle scalafix` but applied correctly by `sbt scalafix` namely `ExplicitResultTypes`.
    I have minimal example [here](https://github.com/jahan01/opensource-issues/tree/master/scalafix_organize_imports).
    Note how `myfile2#myComplexMethod` changes after `sbt scalafix`.

2. Behaviour of certain plugin changes when run via `gradle scalafix`, notably `OrganizeImports`. It produces different ordering with `sbt-scalafix` and `gradle scalafix`. One applied by `sbt scalafix` is correct. Though I have already filed an issue here (liancheng/scalafix-organize-imports#135), I thought it is worth getting your input on why there is different behaviour between gradle and sbt plugin even though scalafix version and semanticDb version are same.

**Comment by bjaglin** [comment]
(1) might be fixed by https://github.com/cosmicsilence/gradle-scalafix/issues/32

**Comment by marcelocenerine** [comment]
@jahan01 thanks for reporting the issues. I will look into them as soon as I can.

**Comment by marcelocenerine** [comment]
@jahan01 , sorry for the lack of update on this. I finally got time to look into this and the good news is that I identified the root cause. I'm working on a fix for this as well as for #32. Please stay tuned

**Comment by marcelocenerine** [comment]
@jahan01  would you mind giving 0.1.6 a try?
